apidoc/Titanium/UI/WebView.yml

```diff
@@ -608,6 +608,13 @@ properties:
     since: "1.8.0"
     default: true
 
+  - name: mixedContentMode
+    summary: If `true`, allows the loading of insecure resources from a secure origin.
+    type: Boolean
+    platforms: [android]
+    since: "7.4.0"
+    default: false
+
```

Comment:
Since we only set this upon creation, we need...
`availability: creation`

The rest looks great. Thanks!

Reply:
And perhaps in the description we should state that iOS users should use the `NSAllowsArbitraryLoads` plist setting.